css/style.css

```diff
@@ -12,7 +12,7 @@
 }
 #title h1{
   font-size: 4rem;
-}
+}/*linter error*/
 #img-div{
```

Comment:
linter error has been found on line 15, there must be an empty space between your css classes for readability reasons, having no space between your classes, makes it hard to read and navigate between them.

Reply:
- [Optional ] Kindly, just like last comment you given below phrased the comments politely, as suggestions instead of orders(above comment a little bit look like an order).